src/lib/components/Accordion/AccordionItem.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import './Accordion.css';
+//import './Accordion.css';
```

Comment:
Should this be commented out?

Reply:
I think it should not be commented and explicitly load the CSS.